frontend/src/components/pages/spells/SpellList.js

```diff
@@ -0,0 +1,63 @@
+import { useState }  from 'react'
+
+import { Grid, Typography } from '@mui/material';
+import { useInfiniteQuery } from '@tanstack/react-query';
+import InfiniteScroll from 'react-infinite-scroller';
+
+import SearchField from '../../ui/SearchField'
+import SpellListItem from './SpellListItem'
+import ListStatusButton from '../../ui/ListStatusButton'
+
+
+
```

Comment:
Can you please remove these empty lines? This also seems to occur in multiple files, please check those again. Just one line of space is enough.